### ISSUE TITLE
Is the project still maintained?

### ISSUE THREAD

**Issue by jerep6** [elastic-coders/serverless-webpack#133]
Hi,

Given the number of pull requests and issues unanswered, I wonder if the project is still maintained.

Do you have more information?

**Comment by StevenACoffman** [comment]
Answer is: sort of?

The original author @thenikso did a fabulous job, but was very up front about not being able to maintain it and asked for someone else to take over. There's been some great work by @HyperBrain and others, but it deserves a fairly sustained amount of effort to keep pace with both Webpack and Serverless. It's possible one or more maintainers will emerge, or it will be moved to the serverless organization and maintained there. 

As for me and my co-workers, we're considering removing the plugin, running webpack, and pointing serverless to the output. 

```bash
NODE_ENV='production' node_modules/.bin/webpack --config webpack.config.js -p
```

**Comment by HyperBrain** [comment]
In my experience this plugin offers the best integration of webpack and Serverless. It would be a loss if it would simply be shut down 😢 Especially building and testing React applications (including server-side rendering) are a pain without the plugin.

Nevertheless, to keep it usable the most important bugs have to be fixed (the ones that really block the use). I could have a look at some of the bugs and try to fix them, but as I'm already quite busy maintaining and contributing to other projects (my ones and Serverless) in my spare time, taking it over completely is currently not an option.

**Comment by StevenACoffman** [comment]
Perhaps the [pull request hack](https://felixge.de/2013/03/11/the-pull-request-hack.html) is called for:

> Whenever somebody sends you a pull request, give them commit access to your project. 

If yourself and others who have contributed pull requests were given access by @domaslasauskas or @thenikso (and also be able to add still more contributrors) it might get this project to a more sustainable place.

**Comment by franciscocpg** [comment]
I agree with @StevenACoffman and I'm glad to help.

**Comment by johnf** [comment]
I also use this heavily and would be happy to help.

**Comment by ricardolpd** [comment]
Our project at this moment is quite dependent on this plugin, and i personally would like to see this plugin moving forward as it is the best integration with with webpack and serverless i have seen. I would be happy to dedicate some of my time weekly for the forseable future making contributions for this project. How would you guys would like to go on about doing this? 
Picking issues and trying to solve it, or just bring in suggested features as we need them and collectively review the pull-requests?

**Comment by StevenACoffman** [comment]
Of the volunteers, @HyperBrain has a well established track record, and at the very least can add a few more maintainers here. @thenikso please add him as a collaborator to this github repo. 
While it is possible for a new github organization to emerge, and this code to be moved there, in the mean time these are the steps remaining:

- [ ] @thenikso adds the npm username maintainers as below <sup id="a1">[1](#f1)</sup>
- [ ] @thenikso adds @HyperBrain as a collaborator to the github repo as below <sup id="a2">[1](#f2)</sup>
- [ ] Potential maintainers provide both their npm userid and github user id here (e.g. `hyperbrain` for npm and `HyperBrain` for github)
- [ ] @HyperBrain can add first few others as they step forward.

<b id="f1">1</b> share npm maintainership like so:. [↩](#a1)

```bash
cd serverless-webpack
npm adduser hyperbrain
npm owner add hyperbrain
```

<b id="f2">2</b> share github maintainership like so:. [↩](#a2)

1. Navigate to the repository on Github you wish to share with your collaborator.
2. Click on the "Settings" tab on the right side of the menu at the top of the screen.
3. On the new page, click the "Collaborators" menu item on the left side of the page.
4. Start typing the new collaborator's GitHub username into the text box.
5. Select the GitHub user from the list that appears below the text box.
6. Click the "Add" button.

**Comment by pmuens** [comment]
Hey guys, great to see efforts to build a community of maintainers around this heavily used plugin.

Just wanted to drop a line that there's also this GitHub org: https://github.com/serverless-community-labs

Maybe worthwhile to contact the developers of the [Simulate plugin](https://github.com/serverless-community-labs/serverless-plugin-simulate) to build up one org where mature plugins are brewed 🍻 .

**Comment by thenikso** [comment]
Wonderful news! I am very sorry for not having being able to sustain the work on this plugin. I'll execute what @StevenACoffman proposes within tomorrow. 

Thank you all for the support and I hope to be back contributing in the near future.

**Comment by thenikso** [comment]
@HyperBrain you should be all set! Please feel free to contact me privately if you need anything concerning this repo.

Also if it would be best for the community to move this repo to a new organization I am all for it.

**Comment by HyperBrain** [comment]
@thenikso Thank you 🥇 . I will contact you if I have any questions or problems moving forward. Now the way should be clear to get the project to an active state again and setup a team to proceed.

Moving it to a new organization would be a goal for the next weeks then. I'll concentrate on the next steps over the upcoming weekend.

**Comment by ricardolpd** [comment]
Guys apologies i have been on holidays with limited internet, if you want to add me, my npm is the same as github.

**Comment by SkippyZA** [comment]
Great to see this up and running again. I landed up doing a fork of this just so I could fix some of the annoying bugs.

I will keep a tab on this and see if I can assist with contributions.

**Comment by HyperBrain** [comment]
@SkippyZA 👍 Please check the recently released version 2.0.0, which contains some of the open PRs (see the [release notes](https://github.com/elastic-coders/serverless-webpack/releases/tag/v2.0.0)). Version 2.1.0 is about to be released soon (here are the [planned features/fixes](https://github.com/elastic-coders/serverless-webpack/milestone/4)).

The major improvements in Version 3.0.0 are [here](https://github.com/elastic-coders/serverless-webpack/milestone/5)

You're welcome to contribute. There are still a lot of issues open that could need some help 😃 . If you have fixed something and think that has to be added too, create an issue for it and (incase you already fixed it) open a PR.
